src/host.cpp

```diff
@@ -561,7 +561,7 @@ namespace rhost {
             }
         }
 
-        extern "C" int ReadConsole(const char* prompt, char* buf, int len, int addToHistory) {
+        extern "C" int R_ReadConsole(const char* prompt, char* buf, int len, int addToHistory) {
```

Comment:
This is needed because it clashes with Win32 `ReadConsole` from kernel32.lib during linking. I'm not sure why static vs dynamic linking of CRT makes a difference here, but either way it's easy to fix by renaming our symbol.